groovy/postInit/chains/ElectronicCircuits.groovy

```diff
@@ -23,6 +23,26 @@ crafting.replaceShaped("gregtech:electronic_circuit_mv", metaitem('circuit.good_
         [metaitem('circuit.electronic'), metaitem('circuit_board.good'), metaitem('circuit.electronic')],
         [metaitem('wireGtSingleCopper'), metaitem('circuit.electronic'), metaitem('wireGtSingleCopper')]])
 
+crafting.addShaped('resistor_wire_anthracite', metaitem('component.resistor') * 2, [
```

Comment:
These recipes contain multiple ingredients which should be using the oredict entries, not the metaitems (wires, dusts, etc)

Reply:
is there a list of oredict entries that I could search?

Reply:
finished changing the metaitem to oredict